netgrasp/config/config.py

```diff
@@ -1,17 +1,18 @@
-from netgrasp.utils import debug
-
 import ConfigParser
 
-config_instance = None
+from netgrasp.utils.email import valid_email_address
+
+DEFAULT_FILENAME = ['/etc/netgrasp.cfg', '/usr/local/etc/netgrasp.cfg', '~/.netgrasp.cfg', './netgrasp.cfg']
```

Comment:
I believe this means we're now defining this default in multiple places: that's very easy to get out of sync. Why aren't you using netgrasp.DEFAULT_CONFIG?